flang/test/Lower/intrinsic-procedures/minval.f90

```diff
@@ -47,3 +47,26 @@ subroutine minval_test3(a,r)
 ! CHECK-DAG:  fir.freemem %[[a13]] : !fir.heap<!fir.array<?xi32>>
 end subroutine
 
+! CHECK-LABEL: func @_QPtest_minval_optional_scalar_mask(
+! CHECK-SAME:  %[[VAL_1:.*]]: !fir.ref<!fir.logical<4>> {fir.optional}
```

Comment:
It looks like a rebase will be needed for these tests. I expect they will fail given the changes to `func` declarations that went in.

Reply:
Indeed :)